### PR TITLE
Inactive Batches Details

### PR DESCRIPTION
**Pull request by xworkzdev6**

### CODE REVIEW

File: src/main/java/com/xworkz/dream/resource/InactiveBatchesController.java
Comment (X-workzDev01) [inline src/main/java/com/xworkz/dream/resource/InactiveBatchesController.java:1]: we don't use Remove response Entity

File: src/main/java/com/xworkz/dream/service/InactiveBatchesServiceImpl.java
Comment (X-workzDev01) [inline src/main/java/com/xworkz/dream/service/InactiveBatchesServiceImpl.java:1]: don't use New Keyword in sheetDto

use stream inside comparator 


remove in lamda expression inside   e charactor

File: src/main/java/com/xworkz/dream/service/util/InactiveBatchesUtil.java
Comment (X-workzDev01) [inline src/main/java/com/xworkz/dream/service/util/InactiveBatchesUtil.java:1]: Refactor predicate

File: src/main/java/com/xworkz/dream/resource/InactiveBatchesController.java
Comment (xworkzdev6) [reply src/main/java/com/xworkz/dream/resource/InactiveBatchesController.java:1]: Removed

File: src/main/java/com/xworkz/dream/service/InactiveBatchesServiceImpl.java
Comment (xworkzdev6) [reply src/main/java/com/xworkz/dream/service/InactiveBatchesServiceImpl.java:1]: Updated

File: src/main/java/com/xworkz/dream/service/util/InactiveBatchesUtil.java
Comment (xworkzdev6) [reply src/main/java/com/xworkz/dream/service/util/InactiveBatchesUtil.java:1]: Updated

**Comment by xworkzdev6** [comment]
Updated code pushed for review!!

**Pull request merged**